documents/MIGRATION_GUIDE.md

```diff
@@ -0,0 +1,954 @@
+service. The Jobs service helps with defining a set of remote operations that can be sent to and run on one or more devices connected
+to AWS IoT.
+
+The Jobs service client provides API similar to API provided by [Client for AWS IoT Device Shadow](#client-for-device-shadow-service).
+First, you subscribe to special topics to get data and feedback from a service. The service client provides API for that.
+After subscribing to all the required topics, the service client can start interacting with the server, for example, update
+the status or request for data. These actions are also performed via client API calls.
+
+For detailed descriptions for the topics used to interact with the Jobs service, see AWS IoT Core documentation for the [Jobs](https://docs.aws.amazon.com/iot/latest/developerguide/jobs-mqtt-api.html) service.
+
+For more information about the service clients, see API documentation for the v2 SDK [Jobs](https://aws.github.io/aws-iot-device-sdk-java-v2/software/amazon/awssdk/iot/iotjobs/IotJobsClient.html).
+
+For code example, see the v2 SDK [Jobs](https://github.com/aws/aws-iot-device-sdk-java-v2/tree/main/samples/Jobs) samples.
+
+### Client for AWS IoT fleet provisioning
+
+[Fleet Provisioning](https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html)
+(also known as Identity Service) is another AWS IoT service that the v2 SDK provides access to. By using AWS IoT fleet provisioning, AWS IoT can generate and securely deliver device
+certificates and private keys to your devices when they connect to AWS IoT for the first time.
+
+The fleet provisioning service client provides an API similar to the API provided by [Client for AWS IoT Device Shadow](#client-for-device-shadow-service).
+First, you subscribe to special topics to get data and feedback from a service. The service client provides API for that.
+After subscribing to all the required topics, the service client can start interacting with the server, for example, update
+the status or request for data. These actions are also performed via client API calls.
+
+For detailed descriptions for the topics used to interact with the Fleet Provisioning service, see AWS IoT Core documentation for [Fleet Provisioning](https://docs.aws.amazon.com/iot/latest/developerguide/fleet-provision-api.html).
+
+For more information about the the Fleet Provisioning service client, see API documentation for the v2 SDK [Fleet Provisioning](https://aws.github.io/aws-iot-device-sdk-java-v2/software/amazon/awssdk/iot/iotidentity/IotIdentityClient.html).
+
+For code examples, see the v2 SDK [Fleet Provisioning](https://github.com/aws/aws-iot-device-sdk-java-v2/tree/main/samples/FleetProvisioning)
+samples.
+
+### Example
+
+It's always helpful to look at a working example to see how new functionality works, to be able to tweak different options,
```

Comment:
Found the bad character (quote)
It's always it should be It's always
The look the same but they are not :D